docs/Doxyfile.in

```diff
@@ -23,7 +23,11 @@ MULTILINE_CPP_IS_BRIEF = YES
 
 TAB_SIZE               = 2
 
-ALIASES                = "semantics=<p></p><i>Semantics:</i>" \
+ALIASES                = "spectrename=@SPECTRE_NAME@" \
```

Comment:
Do we really want a doc substitution for the project name?  Everything, including this PR, hardcodes it, and I don't see a reason to encourage people to do otherwise.

Reply:
Agree and removed